REVIEWING.md

```diff
@@ -0,0 +1,274 @@
+# Review process
+
+Code review is an essential part of our project. It allows us to increase the
+quality of the code while permitting everyone to gain from each other's
+perspective.
+
+## Preparing for a code review
+
+People will spend time reviewing your code, as much as you will spend time
+improving your code to answer their expectations. Easing this process can only
+make everyone happier, and quicken it as a whole.
+
+A few big subbjects, with each a few pieces of advice are key for that:
+
+- General rules for PRs:
+
+  - Address only one subject at a time: Silent fixes are bug fixes included in
+    an unrelated change. They should never make it into the codebase this way,
+    and the fix should be part of a separate PR, for that will make it easy to
+    review/merge quickly and make it independent. This also eases
+    forwardporting/backporting only the necessary bits when relevant.
+
+  - Size of PRs: Big ones are hard to review and will take a long time to
+  actually get merged. PRs can be up to 500 changed lines to facilitate the
+  reviewer's job. Ask yourself this question:
+
+    - Is it possible to make smaller incremental PRs? If that's the case,
+      please go ahead and do it! Commits are atomical changes, and PR molecular
+      changes
+
+    - If not, splitting this into atomical changes (ie multiple commits in the
+      PR) will ease the burden on the reviewer
+
+  - Limit the number of PRs you actually have opened, 3 is a good number, 5 shall
+    be tolerated for a number of very small PRs quick to review and thus merge.
+
+  - Production of PRs and Review of PRs should observe a simple ratio. For each
+    PR produced and merged, you should be participating in the process of
+    merging two other PRs as a reviewer. This will allow to produce as many PRs
+    as we merge, and keep the number of open PRs stable
+
+- Documentation can go a long way for reviewers and knowledge transfer:
+
+  - JSDoc for an API brings a lot to the  table, and eases the review
```

Comment:
This was written at a time when we only ever did JS, aside from a few python scripts that were handled by a default `pylint`, the strictest linter when it comes to python code.